.flake8

```diff
@@ -8,7 +8,9 @@ exclude =
   python/build/
   python/.eggs/
   python/ray/_private/thirdparty/*
+  release/nightly_tests/dataset/inference.py
```

Comment:
We can remove this once we can move the model file back to its own file.

Reply:
Remove this